client/cmdhfmfu.h

```diff
@@ -21,7 +21,8 @@ typedef struct {
 	uint8_t	pack[2];
 	uint8_t tbo1[1];
 	uint8_t	signature[32];	
-	//uint8_t	counter[3];
+	uint8_t	counter[3][3];
```

Comment:
Isn't counters 24bits?  uint32_t[3] ?
Or keep it even simpler,   Counter1,  Counter2, Counter3  ?

I guess this messes up old dumps though when reading them.   One of the main reasons I didn't fiddle with this part ...

Reply:
Yes, 3 counters of 3 bytes each. Thus the double array.